pages/doc/wavefront_release_notes.md

```diff
@@ -13,6 +13,10 @@ This page lists new and updated features for the Tanzu Observability by Wavefron
 * For the latest changes and releases of our **Integrations**, see the [Integrations Release Notes](integrations_new_changed.html).
 * For **Observability for Kubernetes**, go to the [release notes for Wavefront Collector for Kubernetes GitHub repository](https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes/releases).
 
+## 2023-03.x Release Notes
+
+* **Alert Notifications Update**: If your **PagerDuty** [alert target](webhooks_alert_notification.html) is integrated with Slack, the alert notifications in Slack are now more extensive. They now show the complete alert notification summary, which can be up to 1,000 characters.
```

Comment:
Maybe remove now from this sentence: They now show the complete alert notification summary, which can be up to 1,000 characters.